sympy/solvers/solvers.py

```diff
@@ -2046,7 +2047,7 @@ def update(determined, solution):
             bestsol = thissol
         return bestsol
 
-
+@deprecated(useinstead="linsolve", issue=10001, deprecated_since_version="0.7.6")
```

Comment:
This should be 0.7.7 (actually 1.0, but we'll have to change all instances of the former to the latter at some point anyway).